share/external/0readme.md

```diff
@@ -0,0 +1,4 @@
+# Extract the Drogon bundle '24.3.0_external.tar.gz'
```

Comment:
Just an idea, but if you named this file README.md like the repository "front page", if you click into the sub directory on GitHub it _should_ display it beneath the file listing like it does there. Possible I am wrong, but I have seen some repositories do this

Reply:
You're right, I have changed the name now.